### PR TITLE
test(driver): TypeScript RESTful publish and add package test to CI

### PR DESCRIPTION
**Pull request by xleili**

### CODE REVIEW

File: src/connector/TypeScript-REST/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "td2.0-rest-connector",
-  "version": "1.0.0",
+  "version": "1.0.3",
```
Comment (sangshuduo) [inline src/connector/TypeScript-REST/package.json:3]: why bump to 1.0.3?

**Review by sangshuduo — APPROVED** [review]
LGTM

File: src/connector/TypeScript-REST/package.json (hunk 1 above)
Comment (xleili) [reply src/connector/TypeScript-REST/package.json:3]: version < 1.0.3  has critical bug. Fix those bug in current version.

**Comment by sangshuduo** [comment]
need pass CI first

**Pull request merged**